documentation.html

```diff
@@ -73,7 +73,7 @@ <h4 class="mk-heading mk-heading--md">Bare metal management pods</h4>
                     <p>The implementation will focus on using Ironic as its first implementation of the Bare Metal Management Pods, but aims to keep this as an implementation detail under the hood such that alternatives could be added in the future if the need arises.</p>
                      <p>For more information about the choice to use Ironic, see the <a href="https://github.com/metal3-io/metal3-docs/blob/master/design/use-ironic.md">use-ironic design document</a>. </p>
                 </div>
-                <img class="mk-infograph__server" src="assets/images/infograph-stack.svg" alt="stacked computers" />
+                <img class="mk-infograph__server" src="assets/images/infograph-stack.svg" alt="" />
```

Comment:
This was used to avoid issues when validating the website, is there any problem if it's left with the alternate text?